.gitignore

```diff
@@ -27,4 +27,5 @@ nuget.exe
 project.lock.json
 /.vs/
 .build/
-.testPublish/
\ No newline at end of file
+.testPublish/
+launchSettings.json
```

Comment:
I'd rather we were consistent about checking them in so .gitignore files could be more consistent across repos.